src/util/popup.ts

```diff
@@ -38,15 +38,15 @@ export class PopupService<T> {
       this._viewContainerRef.remove(this._viewContainerRef.indexOf(this._windowRef.hostView));
       this._windowRef = null;
 
-      if (this._contentRef.viewRef) {
+      if (this._contentRef && this._contentRef.viewRef) {
```

Comment:
We can use the optional chaining operator:
``` 
if (this._contentRef?.viewRef) {
```
(I thought about this right now, I maybe forgot this idea in previous comments, to be checked !)

Reply:
Oh yes, apparently, it is available starting from typescript `3.7.x` (which we got with Angular 9) and looks like this:

```ts
if (a.b?.c) {
}

// compiles to

if ((_a = a.b) === null || _a === void 0 ? void 0 : _a.c) {
}
```

Will do an additional pass through the code for this one.